prow/cmd/autobump/autobump.sh

```diff
@@ -128,16 +128,37 @@ create-gerrit-pr() {
   git config http.cookiefile "${creds}"
   git remote add upstream "${GERRIT_HOST_REPO}"
 
-	change="$( echo "${GERRIT_HOST_REPO}; ${PROW_INSTANCE_NAME}; ${old_version}" | git hash-object --stdin)"
+	change_id="$(get-change-id)"
```

Comment:
Not related to this PR, but it looks the indentation in this file is pretty inconsistent...

Reply:
Yeah looks like there is a mix of tabs and spaces. I'll fix it next time I touch it.